src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectionHandler.ts

```diff
@@ -13,7 +14,7 @@ export class DefaultReconnectionHandler implements ReconnectionHandler {
   constructor(logger: Logger, overrideDisplay?: ReconnectDisplay, reconnectCallback?: () => Promise<boolean>) {
     this._logger = logger;
     this._reconnectionDisplay = overrideDisplay;
-    this._reconnectCallback = reconnectCallback || (() => window['Blazor'].reconnect());
+    this._reconnectCallback = reconnectCallback || (() => (Blazor.reconnect as any)());
```

Comment:
this._reconnectCallback = reconnectCallback || (() => (Blazor?.reconnect as any)());

Reply:
Don't think that will change anything?

Reply:
No, it doesn't.